api/controllers/WebActionsController.js

```diff
@@ -127,6 +128,77 @@ module.exports = {
     }
   },
 
+  splash: function(req, res) {
+    let { EMAIL, FNAME, PHONE, group } = req.body;
+    let memberHash = crypto
```

Comment:
This seems right. But just wanted to check if you've verified that this returns the hash Mailchimp would expect. Like, in here (https://developer.mailchimp.com/documentation/mailchimp/guides/manage-subscribers-with-the-mailchimp-api/) they mention `urist.mcvankab@freddiesjokes.com` should equal `62eeb292278cc15f5817cb78f7790b08`.

Reply:
Checked and matched also used `passwordgenerator.net` to check other emails.  👍